sqld/src/replication/replica/hook.rs

```diff
@@ -11,17 +9,53 @@ use sqld_libsql_bindings::{ffi::types::XWalFrameFn, wal_hook::WalHook};
 use crate::replication::frame::{Frame, FrameBorrowed};
 use crate::replication::{FrameNo, WAL_PAGE_SIZE};
 
-use super::meta::WalIndexMeta;
 use super::snapshot::TempSnapshot;
 
+// Those are custom error codes returned by the replicator hook.
+pub const SQLITE_EXIT_REPLICATION: c_int = 200;
```

Comment:
nitpick: let's use `LIBSQL_` or even `SQLD_`, to be explicit that it isn't actually an SQLite error code.